pkgs/applications/radio/soapysdr/default.nix

```diff
@@ -24,12 +25,12 @@ in stdenv.mkDerivation {
   };
 
   nativeBuildInputs = [ cmake makeWrapper pkgconfig ];
-  buildInputs = [ libusb ncurses numpy python swig2 ];
+  buildInputs = [ libusb ncurses ]
+    ++ lib.optionals usePython (with python.pkgs; [ numpy python swig2 ]);
```

Comment:
Avoid using withPackages when bindings are provided in `python-packages.nix`, unless you know for sure they won't retain a dependency on that build-time environment.

Reply:
I put numpy in `propagatedBuildInputs`. That should be cleaner now.